src/sonic-py-common/sonic_py_common/port_util.py

```diff
@@ -0,0 +1,196 @@
+"""
+Bridge/Port mapping utility library.
```

Comment:
If these files are simply moved from another repo, let's separate them into a standalone PR, not to mix with other changes not of the same nature.